DBSCE/KeyGeneration.md

```diff
@@ -0,0 +1,91 @@
+### Local key helper on Windows
```

Comment:
I'm not sure I understand organization of this document. It contains pieces of Windows specifics, as well as common parts. Logic of this document is not clear for me.

Reply:
Moved all common parts to Overview.